openstack/keystone/templates/seed-2faproxy.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if and (index .Values "2fa" "enabled") (index .Values "2fa" "seed_user") }}
+apiVersion: "openstack.stable.sap.cc/v1"
+kind: OpenstackSeed
+metadata:
+  name: {{ .Release.Name }}-2faproxy-seed
+
+spec:
+  requires:
+    - {{ .Values.global.keystoneNamespace }}/keystone-seed
+    - monsoon3/domain-default-seed
+    - monsoon3/domain-ccadmin-seed
+
+  domains:
+    - name: {{ index .Values "2fa" "openstack" "OS_DOMAIN_NAME" | required ".Values.2faproxy.openstack.DOMAIN_NAME is missing" }}
```

Comment:
it seems that the error message is not leading us to the right path. Shouldn't it be `.Values.2fa....`?

Reply:
Thanks for spotting this!